service_mesh/v2x/ossm-threescale-webassembly-module.adoc

```diff
@@ -0,0 +1,36 @@
+[id="ossm-threescale-webassembly-module"]
+= The 3scale WebAssembly module
+include::modules/ossm-document-attributes.adoc[]
+:context: ossm-threescale-webassembly-module
+
+toc::[]
+
+The following content describes how to configure and run the `threescale-wasm-auth` module for the integration of 3scale API Management 2.11 or later with the {ProductName} 2.1.0 or later.
+
+.An introduction to the `threescale-wasm-auth` module
+
+The `threescale-wasm-auth` module is a link:https://webassembly.org[WebAssembly] module that uses a set of interfaces, known as an application binary interfaces (_ABI_). This is defined by the link:https://github.com/proxy-wasm/spec[_Proxy-WASM_] specification to drive any piece of software that implements the _ABI_ so it can authorize HTTP requests against 3scale.
+
+As an _ABI_ specification, _Proxy-WASM_ defines the interaction between a piece of software named host and another named _module_, _program_, or _extension_. The host exposes a set of services used by the module to perform a task, and in this case, to process proxy requests.
```

Comment:
"defines the interaction between a piece of software named host"

Should we be italicizing "host" here?

Reply:
Yes, I believe so.